drivers/qemu/driver.go

```diff
@@ -344,6 +346,19 @@ func isAllowedImagePath(allowedPaths []string, allocDir, imagePath string) bool
 	return false
 }
 
+func isAllowedDriveInterface(driveInterface string) bool {
+	// hardocoded list of drive interfaces, Qemu currently supports
+	var interfaceList = []string{"ide", "scsi", "sd", "mtd", "floppy", "pflash", "virtio", "none"}
+
+	for _, ai := range interfaceList {
```

Comment:
This isn't in a hot path, but no reason to allocate this whole list for each function call:

```suggestion
// hardcoded list of drive interfaces, Qemu currently supports
var allowedDriveInterfaces = []string{"ide", "scsi", "sd", "mtd", "floppy", "pflash", "virtio", "none"}

func isAllowedDriveInterface(driveInterface string) bool {
	for _, ai := range allowedDriveInterfaces {
```